app/controllers/cart_controller.rb

```diff
@@ -13,6 +13,7 @@ def add_item
   end
 
   def show
+    @user = current_user
```

Comment:
Is @user used in the cart show html?

Reply:
I must have left it in there when I was trying to figure out the logic, good catch!

Reply:
Can you update PR